tests/kafkatest/services/security/templates/jaas.conf

```diff
@@ -55,6 +55,7 @@ KafkaServer {
     useKeyTab=true
     storeKey=true
     keyTab="/mnt/security/keytab"
+    refreshKrb5Config=true
```

Comment:
This change is needed because of an issue in GraalVm native-image. 
I raised this in their community slack channel.
This is a workaround for now.
Conversation thread: https://graalvm.slack.com/archives/CN9KSFB40/p1700246576831259